assets/app/scripts/directives/podStatusChart.js

```diff
@@ -48,10 +48,11 @@ angular.module('openshiftConsole')
           type: "donut",
           bindto: '#' + $scope.chartId,
           donut: {
-            width: 10,
+            expand: false,
```

Comment:
It removes the effect that grows the arc on hover, which looked weird for the no pods case. We also don't handle clicks on the arcs anyway. The tooltip still displays.